calibration_date: '2025-06-11T12:37:23'
camera_matrix:
- - 2057.119835773729
  - 0.0
  - 1287.3560963952127
- - 0.0
  - 2057.119835773729
  - 1035.3996042751426
- - 0.0
  - 0.0
  - 1.0
distortion_coefficients:
- - 0.12038549159180839
  - -0.16288890577881324
  - 0.010856869622826244
  - 0.0010147680490626516
  - 54.691172800813185
  - 0.0
  - 0.0
  - 53.47795733522252
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
focal_length:
  fx: 2057.119835773729
  fy: 2057.119835773729
image_size:
- 2592
- 1944
principal_point:
  cx: 1287.3560963952127
  cy: 1035.3996042751426
reprojection_error_mean: 1.1049442111566197
valid_images_count: 36
